tests/browser/globals.d.ts

```diff
@@ -10,6 +10,9 @@ declare global {
     const popupUtils: {
         click: (selector: string) => Promise<void>;
         exists: (selector: string) => Promise<boolean>;
-        getBoundingRect: (selector: string) => Promise<{left: number; top: number; width: number; height: number}>;
```

Comment:
I removed `popupUtils.getBoundingRect` because it is not used.

Reply:
Seems good.